PerfTools/AllocMonitor/interface/mea_AllocMap.h

```diff
@@ -0,0 +1,98 @@
+#ifndef PerfTools_AllocMonitor_mea_AllocMap_h
+#define PerfTools_AllocMonitor_mea_AllocMap_h
+// -*- C++ -*-
+//
+// Package:     PerfTools/AllocMonitor
+// Class  :     AllocMap
+//
+/**\class mea_AllocMap mea_AllocMap.h "PerfTools/AllocMonitor/interface/mea_AllocMap.h"
+
+ Description: [one line class summary]
+
+ Usage:
+    <usage>
+
+*/
+//
+// Original Author:  Christopher Jones
+//         Created:  Fri, 11 Oct 2024 19:15:46 GMT
+//
+
+// system include files
+#include <vector>
+#include <cassert>
+
+// user include files
+
+// forward declarations
+
+namespace moduleEventAlloc {
```

Comment:
The `moduleAlloc_*` files use `edm::service::moduleAlloc` namespace, so how about using `edm::service::moduleEventAlloc` here?

Reply:
done